irc/histserv.go

```diff
@@ -15,6 +15,12 @@ import (
 	"github.com/ergochat/ergo/irc/utils"
 )
 
+const (
+	canDeleteAny  = iota // User is allowed to delete any message (for a given channel/PM)
```

Comment:
Add a type for these, e.g.

https://github.com/ergochat/ergo/blob/bf33fba33a7a5326510dd643ddf92a57cfaec841/irc/config.go#L71-L78